src/Infrastructure/LeanCode.Kratos/KratosConfig.cs

```diff
@@ -0,0 +1,10 @@
+namespace LeanCode.Kratos;
+
+public sealed record class KratosConfig(
+    string AdminEndpoint,
```

Comment:
`Uri`s?

Reply:
Fair, I went with `string`s because this is what `*Api` clients accept but I can change that.

Reply:
Actually we can have both if we dump the `Config` class entirely :upside_down_face: